src/processor.ts

```diff
@@ -17,11 +17,13 @@ const transformRecord = ({ recordId, data }: EventRecord): ResultRecord => {
     const event = JSON.parse(Buffer.from(data, "base64").toString());
     const { time, data: { raw } } = struct(event, OCSEvent); // prettier-ignore
     const datetime = localFromISO(time);
+    // Mimic the timestamp prepended by the old OCS.LogUploader from RTR
+    const timestampedRaw = `${datetime.toFormat("MM/dd/yy,HH:mm:ss")},${raw}`;
```

Comment:
question: are there downstream processors that expect this format (instead of ISO 8601) already?

Reply:
Unknown; I don't think we have a great idea of exactly how existing consumers are using this data. But if we didn't include something here, it would almost certainly break their code, since the data has no "headers" and has to be indexed positionally. Thinking of this whole project as a pure "refactor", I'd prefer to replicate the existing logic exactly, then consider further changes as separate tasks (which might require some chats with consumers).